arches/app/models/graph.py

```diff
@@ -538,6 +538,12 @@ def save(self, validate=True, nodeid=None):
     def delete(self):
         if self.is_editable() is True:
             with transaction.atomic():
+                try:
+                    editable_future_graph = models.GraphModel.objects.get(source_identifier=self.graphid)
+                    editable_future_graph.delete()
+                except models.GraphModel.DoesNotExist:
+                    pass  # no editable future graph to delete
```

Comment:
nit: Can this try/except be refactored to use `filter` and checking the len of the return? I feel that as a team we lean a bit heavy on try/excepts, to the point I'm thinking of having a quick brownbag about it 😄